core/src/main/java/hudson/node_monitors/AbstractNodeMonitorDescriptor.java

```diff
@@ -227,7 +227,7 @@ protected boolean markOnline(Computer c) {
      *      or the computer already marked offline.)
      */
     protected boolean markOffline(Computer c, OfflineCause oc) {
-        if(isIgnored() || c.isTemporarilyOffline()) return false; // noop
+        if(isIgnored() || c.isOffline()) return false; // noop
```

Comment:
Pretty sure this is wrong given what it does.

Reply:
Yes, you are right. It is a little confusing to mark something as deprecated to warn others not to use it.